### PR TITLE
feat: add cli:versions:inspect command

### PR DESCRIPTION
**Pull request by mdonnalley**
### What does this PR do?

Adds a `cli:versions:inspect` command so that we can quickly verify that all install paths have the same version

```
~/code/plugin-release-management [mdonnalley/versions-inspect] : bin/run cli:versions:inspect --help
inspect the CLI version across all install paths

USAGE
  $ sfdx cli:versions:inspect -c <string> -l <string> [-d <string>] [-s] [--json] [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]

OPTIONS
  -c, --channels=legacy|stable|stable-rc|latest|latest-rc                           (required) the channel you want to inspect (for achives, latest and latest-rc are
                                                                                    translated to stable and stable-rc. And vice-versa for npm)

  -d, --dependencies=dependencies                                                   glob pattern of dependencies you want to see the version of

  -l, --locations=archive|npm                                                       (required) the location you want to inspect

  -s, --salesforce                                                                  show versions of salesforce owned dependencies

  --json                                                                            format output as json

  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for this command invocation

EXAMPLES
  sfdx cli:versions:inspect -l archive -c stable
  sfdx cli:versions:inspect -l archive -c stable-rc
  sfdx cli:versions:inspect -l archive npm -c stable
  sfdx cli:versions:inspect -l archive npm -c latest
  sfdx cli:versions:inspect -l archive npm -c latest latest-rc
  sfdx cli:versions:inspect -l archive npm -c stable stable-rc
  sfdx cli:versions:inspect -l npm -c latest --salesforce
  sfdx cli:versions:inspect -l npm -c latest -d @salesforce/core
  sfdx cli:versions:inspect -l npm -c latest -d @salesforce/**/ salesforce-alm
  sfdx cli:versions:inspect -l npm -c latest -d chalk -s

```

```
~/code/plugin-release-management [mdonnalley/versions-inspect] : bin/run cli:versions:inspect -l archive npm -c stable-rc --salesforce
Working Directory: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-darwin-x64.tar.gz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-darwin-x64.tar.gz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-darwin-x64.tar.xz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-darwin-x64.tar.xz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-arm.tar.gz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-linux-arm.tar.gz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-arm.tar.xz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-linux-arm.tar.xz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-x64.tar.gz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-linux-x64.tar.gz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-x64.tar.xz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-linux-x64.tar.xz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x64.tar.gz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-win32-x64.tar.gz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x64.tar.xz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-win32-x64.tar.xz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x86.tar.gz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-win32-x86.tar.gz... done
Downloading: https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x86.tar.xz... done
Unpacking: /var/folders/cf/zyy_gbt12194b_04xf7mlf580000gp/T/cli_inspection/unpacked/sfdx-win32-x86.tar.xz... done
Installing: sfdx-cli@latest-rc... done

https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-darwin-x64.tar.gz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-darwin-x64.tar.xz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-arm.tar.gz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-arm.tar.xz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-x64.tar.gz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-x64.tar.xz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x64.tar.gz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x64.tar.xz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x86.tar.gz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-win32-x86.tar.xz: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.18.6
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0
https://www.npmjs.com/package/sfdx-cli/v/7.88.3: 7.88.3
  salesforce-alm: 51.0.1
  salesforcedx: 51.0.2
  @salesforce/apex-node: 0.1.4
  @salesforce/bunyan: 2.0.0
  @salesforce/command: 3.1.0
  @salesforce/core: 2.19.0
  @salesforce/kit: 1.4.0
  @salesforce/lazy-require: 0.4.0
  @salesforce/plugin-alias: 1.1.7
  @salesforce/plugin-analytics: 1.12.1
  @salesforce/plugin-apex: 0.1.4
  @salesforce/plugin-auth: 1.4.8
  @salesforce/plugin-config: 1.2.4
  @salesforce/plugin-custom-metadata: 1.0.11
  @salesforce/plugin-generator: 1.1.5
  @salesforce/plugin-limits: 1.0.3
  @salesforce/plugin-schema: 1.0.3
  @salesforce/plugin-telemetry: 1.1.1
  @salesforce/plugin-templates: 51.1.0
  @salesforce/plugin-user: 1.1.0
  @salesforce/require-analytics: 0.7.3
  @salesforce/schemas: 1.0.3
  @salesforce/sfdx-plugin-lwc-test: 0.1.7
  @salesforce/sfdx-trust: 3.6.0
  @salesforce/source-deploy-retrieve: 1.1.17
  @salesforce/telemetry: 2.0.5
  @salesforce/templates: 51.1.0
  @salesforce/ts-types: 1.5.0

All archives match? true
All archive@stable-rc versions match? true
npm@latest-rc and all archive@stable-rc versions match? true
```

### CODE REVIEW

File: src/commands/cli/versions/inspect.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  public static readonly description = messages.getMessage('description');
+
+  public static readonly flagsConfig: FlagsConfig = {};
+
+  public async run(): Promise<Info[]> {
+    const tarDir = path.resolve(path.join('.', 'tmp', 'tar'));
+    await fs.mkdirp(tarDir);
+
+    const results: Info[] = [];
+    for (const insatllerPath of INSTALLER_PATHS) {
```
Comment (shetzel) [inline src/commands/cli/versions/inspect.ts:53]: nit: `insatllerPath` --> `installerPath`

File: src/commands/cli/versions/inspect.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    this.ux.log();
+    for (const result of results) {
+      this.ux.log(`${result.origin}: ${green(result.version)}`);
+    }
+    this.ux.log();
+    const archivesMatch = new Set(results.filter((r) => r.type === 'archive').map((r) => r.version)).size === 1;
+    this.ux.log(`All Archives Match? ${archivesMatch ? green(archivesMatch) : red(archivesMatch)}`);
+    const allMatch = new Set(results.filter((r) => r.tag === 'latest').map((r) => r.version)).size === 1;
+    this.ux.log(`All Latest Versions Match? ${allMatch ? green(allMatch) : yellow(allMatch)}`);
+    return results;
```
Comment (WillieRuemmele) [inline src/commands/cli/versions/inspect.ts:102]: does this make sense? 
```
https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz: 7.82.0
npm (latest): 7.86.3
npm (latest-rc): 7.87.0

All Archives Match? true
All Latest Versions Match? false
```
that's showing three versions. Is this just verifying that all of the archives match each other, not necessarily matching to npm's `latest`? Shouldn't there ever only be 2 versions?

**Review by shetzel — APPROVED** [review]
Super awesome!  Thanks for doing this!

File: src/commands/cli/versions/inspect.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import { exec } from 'shelljs';
+import { FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { fs, Messages } from '@salesforce/core';
+import { green, red, cyan, yellow } from 'chalk';
+import { NpmPackage, PackageJson } from '../../../package';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'cli.versions.inspect');
+
+const PATH_1 = 'https://developer.salesforce.com/media/salesforce-cli/sfdx-cli/channels/stable';
```
Comment (peternhale) [inline src/commands/cli/versions/inspect.ts:18]: Will these paths change once Phillipe is done with his S3 work?

**Review by amphro — COMMENTED** [review]
First, I think this sort of thing should be verified by the builds. We already have the artifacts we are pushing up to S3. Why are those jobs passing when they clearly didn't copy over the new versions? Second, this won't work with Philipe's changes.

**Review by peternhale — APPROVED** [review]

**Review by peternhale — APPROVED** [review]

**Review by peternhale — APPROVED** [review]

**Pull request merged**